Source/WebKit/UIProcess/PageClient.h

```diff
@@ -471,7 +471,8 @@ class PageClient : public CanMakeWeakPtr<PageClient> {
     virtual void updateTextIndicator(Ref<WebCore::TextIndicator>) = 0;
     virtual void clearTextIndicator(WebCore::TextIndicatorDismissalAnimation) = 0;
     virtual void setTextIndicatorAnimationProgress(float) = 0;
-    
+    virtual CALayer* getTextIndicatorInstallationLayer() = 0;
```

Comment:
Nit: https://www.webkit.org/code-style-guidelines/#pointers-non-cpp
```suggestion
    virtual CALayer *getTextIndicatorInstallationLayer() = 0;
```